src/blocks/Libraries/components/LibraryPreview/LibraryPreview.tsx

```diff
@@ -0,0 +1,124 @@
+import chartkitAsset from '../../../../assets/libs/img-lib-chartkit.jpg';
+import dashkitAssetMobile from '../../../../assets/libs/img-lib-dashkit-mobile.jpg';
+import dashkitAsset from '../../../../assets/libs/img-lib-dashkit.jpg';
+import datecomponentsAssetMobile from '../../../../assets/libs/img-lib-datecomponents-mobile.jpg';
+import datecomponentsAsset from '../../../../assets/libs/img-lib-datecomponents.jpg';
+import navigationAssetMobile from '../../../../assets/libs/img-lib-navigation-mobile.jpg';
+import navigationAsset from '../../../../assets/libs/img-lib-navigation.jpg';
+import pageconstructorAssetMobile from '../../../../assets/libs/img-lib-pageconstructor-mobile.jpg';
+import pageconstructorAsset from '../../../../assets/libs/img-lib-pageconstructor.jpg';
+import uikitAssetMobile from '../../../../assets/libs/img-lib-uikit-mobile.jpg';
+import uikitAsset from '../../../../assets/libs/img-lib-uikit.jpg';
+import wysiwygAssetMobile from '../../../../assets/libs/img-lib-wysiwyg-mobile.jpg';
+import wysiwygAsset from '../../../../assets/libs/img-lib-wysiwyg.jpg';
+import {useIsMobile} from '../../../../hooks/useIsMobile';
+import {block, getLibById} from '../../../../utils';
+
+import './LibraryPreview.scss';
+
+const b = block('libraries-library-preview');
+
+export type FeatureItemProps = {
+    id?: string;
+    title: string;
+    description?: string;
+    contentStyle: Record<string, unknown>;
+};
+
+const libIdToAsset: Record<string, {src: string}> = {
+    uikit: uikitAsset,
+    'page-constructor': pageconstructorAsset,
+    navigation: navigationAsset,
+    'markdown-editor': wysiwygAsset,
+    dashkit: dashkitAsset,
+    chartkit: chartkitAsset,
+    'date-components': datecomponentsAsset,
+};
+
+const libIdToAssetMobile: Record<string, {src: string}> = {
+    uikit: uikitAssetMobile,
+    'page-constructor': pageconstructorAssetMobile,
+    navigation: navigationAssetMobile,
+    'markdown-editor': wysiwygAssetMobile,
+    dashkit: dashkitAssetMobile,
+    chartkit: chartkitAssetMobile,
+    'date-components': datecomponentsAssetMobile,
+};
+
+function getBackgroundImage(id?: string, isMobile?: boolean) {
+    const assetsMap = isMobile ? libIdToAssetMobile : libIdToAsset;
+    const asset = id ? assetsMap[id] : undefined;
+    if (asset) {
+        return `url(${asset.src})`;
+    }
+    return undefined;
+}
+
+export const LibraryPreview: React.FC<FeatureItemProps> = ({
+    id,
+    title,
+    description,
+    contentStyle,
+}) => {
+    const isMobile = useIsMobile();
+
+    let starsCount;
+    let isPrimary = false;
+
+    if (id) {
+        const {data: libData, config: libConfig} = getLibById(id);
+
+        if (libData) {
+            starsCount = libData.stars ?? 0;
+        }
+
+        if (libConfig) {
+            isPrimary = libConfig.primary;
+        }
+    }
+
+    const Tag = id ? (Link as any) : 'div';
+    const tagProps = id ? {href: `/libraries/${id}`} : {};
```

Comment:
`/libraries/${id}` -> `getLocaleLink(`/libraries/${id}`)`
for multilang support